vcf-dumper/vcf-dumper-lib/src/test/java/embl/ebi/variation/eva/vcfdump/regionutils/RegionFactoryTest.java

```diff
@@ -88,4 +106,29 @@ public void divideRegionInChunks() {
         regions = regionFactory.divideRegionInChunks("1", 3000, 2500);
         assertTrue(regions.size() == 0);
     }
+
+    @Test
+    public void executeQueryForGettingMinAndMaxCoordinatesWhenTheyAreNotIncludedInTheRegionFilter() throws IOException, IllegalOpenCGACredentialsException {
```

Comment:
The first case is already tested in the method "getRegionsForChromosome". I added those regions to the filter to check that is parsing well the regions filter. I could extract the region query parser code to a different class, but it would be a very small class and I'm not sure if is worth it. I've written this test again, now we test the case 2 with different inputs.